src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 * Updated [Http Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/http.md).
-  * This library can emit either old, new, or both attributes. Users can control
+  This library can emit either old, new, or both attributes. Users can control
```

Comment:
I think we should refer to specific version of semantic conventions here. (Needs change before release)